src/cultionet/models/base_layers.py

```diff
@@ -1,10 +1,83 @@
 import typing as T
-
-from . import model_utils
+import enum
 
 import torch
+import torch.nn.functional as F
 from torch_geometric import nn
 
+from . import model_utils
+from .enums import ResBlockTypes
+
+
+class Swish(torch.nn.Module):
+    def __init__(self, channels: int, dims: int):
+        super(Swish, self).__init__()
+
+        self.sigmoid = torch.nn.Sigmoid()
+        # assert dims in (2, 3)
+        # if dims == 2:
+        #     ones = torch.ones((1, channels, 1, 1))
+        # else:
+        #     ones = torch.ones((1, channels, 1, 1, 1))
+        self.beta = torch.nn.Parameter(torch.ones(1))
+
+    def forward(self, x: torch.Tensor) -> torch.Tensor:
+        return x * self.sigmoid(self.beta * x)
+
+    def reset_parameters(self):
+        torch.nn.init.ones_(self.beta)
+
+
+class SetActivation(torch.nn.Module):
+    def __init__(
+        self,

```

Comment:
nice small class! Now you can just run the followings without run `torch.nn.{activation_type}`
```
SetActivation( activation_type, channels=out_channels, dims=2)
```